op-program/client/boot_test.go

```diff
@@ -40,10 +70,18 @@ func (o *mockBoostrapOracle) Get(key preimage.Key) []byte {
 		return o.b.L2Claim[:]
 	case L2ClaimBlockNumberLocalIndex.PreimageKey():
 		return binary.BigEndian.AppendUint64(nil, o.b.L2ClaimBlockNumber)
+	case L2ChainIDLocalIndex.PreimageKey():
+		return binary.BigEndian.AppendUint64(nil, o.b.L2ChainID)
 	case L2ChainConfigLocalIndex.PreimageKey():
+		if !o.custom {
+			panic("unexpected key")
```

Comment:
Might be worth including the key value in the panic message or at least make the message say it was the chain config index so we can tell which case was hit. Line numbers might be enough but can easily wind up out of sync.